internal/liqonet/tunnel-operator/tunnel-operator.go

```diff
@@ -254,12 +254,7 @@ func (tc *TunnelController) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		return result, nil
 	}
 	clusterID = tep.Spec.ClusterID
-	// We wait for the resource to be ready. The resource is created in two steps, first the spec and metadata fields
-	// then the status field. So we wait for the status to be ready.
-	if tep.Status.Phase != liqoconst.TepReady {
```

Comment:
We should remove the definition of `liqoconts.TepReady`. It is not used anymore.

Reply:
Thanks